src/plotman/plotters/madmax.py

```diff
@@ -649,3 +665,131 @@ def _cli_974d6e5f1440f68c48492122ca33828a98864dfc() -> None:
+@commands.register(version=(3,))
+@click.command()
+# https://github.com/madMAx43v3r/chia-plotter/blob/ecec17d25cd547fa4bb64b2eb7455b831c8a2882/LICENSE
+# https://github.com/madMAx43v3r/chia-plotter/blob/ecec17d25cd547fa4bb64b2eb7455b831c8a2882/src/chia_plot.cpp#L257-L277
+@click.option(
+    "-k",
+    "--size",
+    help="K size (default = 32, k <= 34)",
+    type=int,
+    default=32,
+    show_default=True,
+)
+@click.option(
+    "-x",
+    "--port",
+    help="Network port (default = 8444, chives = 9699, mmx = 11337)",
+    type=int,
+    default=8444,
+    show_default=True,
+)
+@click.option(
+    "-n",
+    "--count",
+    help="Number of plots to create (default = 1, -1 = infinite)",
+    type=int,
+    default=1,
+    show_default=True,
+)
+@click.option(
+    "-r",
+    "--threads",
+    help="Number of threads (default = 4)",
+    type=int,
+    default=4,
+    show_default=True,
+)
+@click.option(
+    "-u",
+    "--buckets",
+    help="Number of buckets (default = 256)",
+    type=int,
+    default=256,
+    show_default=True,
+)
+@click.option(
+    "-v",
+    "--buckets3",
+    help="Number of buckets for phase 3+4 (default = buckets)",
+    type=int,
+    default=256,
+)
+@click.option(
+    "-t",
+    "--tmpdir",
+    help="Temporary directory, needs ~220 GiB (default = $PWD)",
+    type=click.Path(),
+    default=pathlib.Path("."),
+    show_default=True,
+)
+@click.option(
+    "-2",
+    "--tmpdir2",
+    help="Temporary directory 2, needs ~110 GiB [RAM] (default = <tmpdir>)",
+    type=click.Path(),
+    default=None,
+)
+@click.option(
+    "-d",
+    "--finaldir",
+    help="Final directory (default = <tmpdir>)",
+    type=click.Path(),
+    default=pathlib.Path("."),
+    show_default=True,
+)
+@click.option(
+    "-s",
+    "--stagedir",
+    help="Stage directory (default = <tmpdir>)",
+    type=click.Path(),
+    default=None,
+    show_default=True,
+)
+@click.option(
+    "-w",
+    "--waitforcopy",
+    help="Wait for copy to start next plot",
+    type=bool,
+    default=False,
+    show_default=True,
+)
+@click.option(
+    "-p", "--poolkey", help="Pool Public Key (48 bytes)", type=str, default=None
+)
+@click.option(
+    "-c", "--contract", help="Pool Contract Address (62 chars)", type=str, default=None
+)
+@click.option(
+    "-f", "--farmerkey", help="Farmer Public Key (48 bytes)", type=str, default=None
+)
+@click.option(
+    "-G", "--tmptoggle", help="Alternate tmpdir/tmpdir2", type=str, default=None
+)
+@click.option(
+    "-D",
+    "--directout",
+    help="Create plot directly in finaldir (default = false)",
+    type=bool,
+    default=False,
+)
+@click.option(
+    "-Z",
+    "--unique",
+    help="Make unique plot (default = false)",
+    type=bool,
+    default=False,
+)
+@click.option(
+    "-K",
+    "--rmulti2",
+    help="Thread multiplier for P2 (default = 1)",
+    type=int,
+    default=1,
+)
+def _cli_ecec17d25cd547fa4bb64b2eb7455b831c8a2882() -> None:
```

Comment:
What are these?

Reply:
The `ecec17d...` is the commit hash of madmax from which this was translated.  It matches the link at the top of the command definition.  Presently we just use the latest one according to the version provided in the code like `@commands.register(version=(3,))`.  In the future we could tie some version detection code to make it pick the 'right' command definition for that version.